src/taco-remote/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "taco-remote",
     "description": "Module for the remotebuild package to build, run, and debug iOS apps created using Apache Cordova.",
-    "version": "1.2.1-dev",
+    "version": "1.3.1-dev",
```

Comment:
This is correct, except the master branch here is one step ahead of when these versioning changes happened: the v1.2.0 branch has taco-remote version 1.3.0 in it. Master branch has "the next version" of taco-remote going forwards, so it has a larger version number, which will be corrected when we prepare the next build to publish (possibly remaining as a patch bump as this represents, possibly a minor or major bump if we make appropriate changes).